ocaml-lsp-server/test/e2e/__tests__/__snapshots__/workspace-symbol.test.ts.snap

```diff
@@ -0,0 +1,1485 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`workspace/symbol returns symbols from workspace 1`] = `
```

Comment:
This test should be cleaned up so the output is much smaller. Perhaps a different workspace? Or post process the result to only include the important stuff?

Reply:
https://github.com/ocaml/ocaml-lsp/pull/398/commits/1e26f5707299c507865e063ce78184508b09e4a6 does that look better?

I also switched over to using inlineSnaptshot in https://github.com/ocaml/ocaml-lsp/pull/398/commits/1b8a17b51cf439fc6c68ca3777fe33308f11162b since that's what is being used elsewhere

Reply:
Looks fine to me.